src/Middleware/HeaderPropagation/src/DependencyInjection/HeaderPropagationHttpClientBuilderExtensions.cs

```diff
@@ -11,6 +12,9 @@ public static class HeaderPropagationHttpClientBuilderExtensions
         /// <summary>
         /// Adds a message handler for propagating headers collected by the <see cref="HeaderPropagationMiddleware"/> to a outgoing request.
         /// </summary>
+        /// <remarks>
+        /// The headers will be named using the value specified in <see cref="HeaderPropagationEntry.CapturedHeaderName"/>.
```

Comment:
Does this need to go into more details?

Reply:
I think this comment could be left out. It doesn't really explain what's going on in a clear way.

Reply:
Actually I would change it to say something.

```
When using this method, all configured headers will be applied to outgoing HTTP requests.
```